### DISCUSSION TITLE
ideavimrc not being sourced by clion?

### DISCUSSION THREAD

**Discussion by IssaAboudi** [JetBrains/ideavim#694] [Q&A]
Hello! I've made some bindings in ~/.ideavimrc but I'm not seeing any of them take effect in Clion. i have the plugin installed and I've restarted the editor multiple times.

I have very simple commands in there:

nnoremap gt :action GotoNextSplitter<CR>
nnoremap gT :action GotoPreviousSplitter<CR>

**Comment by citizenmatt** [comment]
Hi. Sorry for the long delay - did you get this sorted? I think these maps are missing a `<CR>`, so they should be:

```
nnoremap gt :action GotoNextSplitter<CR>
nnoremap gT :action GotoPreviousSplitter<CR>
```

Although it would be better to map them with the `<Action>` syntax, which usually works better from mappings:

```
nmap gt <Action>(GotoNextSplitter)
nmap gT <Action>(GotoPreviousSplitter)
```

(Note the `nmap` and not `nnoremap` - this is required for the `<Action>` syntax)
